substrate/client/consensus/babe/rpc/Cargo.toml

```diff
@@ -16,7 +16,7 @@ workspace = true
 targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
-jsonrpsee = { version = "0.22", features = ["client-core", "macros", "server"] }
+jsonrpsee = { version = "0.22", features = ["client-core", "server-core", "macros"] }
```

Comment:
If the `server-core` feature fails to compile before 0.22.5, maybe you need to point the version to 0.22.5.

Reply:
Yeah, for downstream users, it's neceessary